DominoSwissWeatherstation/module.php

```diff
@@ -66,9 +91,38 @@ public function ReceiveData($JSONString) {
 	
 	}
 
+	private function GetStepFromWindValue($value) {
+		$index = -1;
+		foreach (self::VALID_WIND_VALUES as $key => $val) {
+			if ($val == $value) {
+				$index = $key;
+				break;
+			}
+		}
+		return $index;
+	}
+
+	private function GetStepFromLightValue($value) {
+		$index = -1;
+		foreach (self::VALID_LIGHT_VALUES as $key => $val) {
+			if ($val == $value) {
+				$index = $key;
+				break;
+			}
+		}
+		return $index;
+	}
 	
+	private function GetSimpleWindValue($step) {
```

Comment:
Bit of an overkill to pack this in a separate function.
also complicates things a bit more due to the naming differences.

Reply:
Eh, I see your point, but I like that it's descriptive when reading the code above. But you're right regarding naming confusion. So I just deleted the old functions :)